lib/app/features/user/providers/follow_list_provider.r.dart

```diff
@@ -57,10 +57,13 @@ Future<FollowListEntity?> currentUserFollowList(Ref ref) async {
 Future<List<String>> currentUserFollowListWithMetadata(
   Ref ref, {
   int limit = 50,
+  bool hasLimit = true,
```

Comment:
It looks bad to have both `int limit` and `bool hasLimit ` params in the same function, maybe just make the limit nullable and pass `null` when it is not needed (if we pass null explicitly, default value won't be used). wdyt?
```
int? limit = 50,
....
final followedPeople = (limit != null ? followeList?.take(limit) : followeList) ?? [];
```

then instead of 
```
hasLimit: hasLimit,
```

just pass it like
```
limit: null
```